ui_tests/pages/groups_page.py

```diff
@@ -0,0 +1,292 @@
+import time
+
+import pages.helper as helper
+from logger.logger import Logger
+from selenium.webdriver.common.by import By
+from webdriver.webriver_factory import WebDriverFactory
+
+logger = Logger().get_logger()
+driver = WebDriverFactory.get_driver()
+
+
+class GroupsPage:
+    def checkIfGroupsTableIsDisplayed(self):
+        logger.info("Check if groups page is displayed")
+        groups_table_xpath = "//div[@data-test='sc4snmp:group-table']"
+        groups_container = driver.find_element(By.XPATH, groups_table_xpath)
+        return groups_container.is_displayed()
+
+    def click_add_new_group_button(self):
+        logger.info(f"Click add new group button")
+        add_group_button_xpath = "//button[@data-test='sc4snmp:new-item-button']//span"
+        add_grp_btn = driver.find_element(By.XPATH, add_group_button_xpath)
+        add_grp_btn.click()
+        time.sleep(1)
+
+    def set_group_name(self, group_name):
+        logger.info(f"Set group name: {group_name}")
+        add_grp_input = self._get_group_name_input()
+        add_grp_input.send_keys(group_name)
+
+    def _get_group_name_input(self):
+        add_group_input_xpath = (
+            "//div[@data-test='sc4snmp:form:group-name-input']//span//input"
+        )
+        add_grp_input = driver.find_element(By.XPATH, add_group_input_xpath)
+        return add_grp_input
+
+    def click_submit_button_for_add_group(self):
+        logger.info(f"Click submit button")
+        add_group_button_xpath = (
+            "//button[@data-test='sc4snmp:form:submit-form-button']"
+        )
+        add_grp_btn = driver.find_element(By.XPATH, add_group_button_xpath)
+        add_grp_btn.click()
+        time.sleep(5)  # wait for group to be shown on the list
+
+    def click_submit_button_for_add_device(self):
+        self.click_submit_button_for_add_group()
+
+    def click_cancel_button_for_add_device(self):
+        logger.info(f"Click cancel button")
+        cancel_button_xpath = "//button[@data-test='sc4snmp:form:cancel-button']"
+        cancel_btn = driver.find_element(By.XPATH, cancel_button_xpath)
+        cancel_btn.click()
+
+    def check_if_groups_is_on_list(self, group_name):
+        logger.info(f"Checking if group is configured (is on list)")
+        group_entry_on_list_xpath = "//div[@data-test='sc4snmp:group']//p"
+        groups_entries = driver.find_elements(By.XPATH, group_entry_on_list_xpath)
+        for el in groups_entries:
+            logger.info(f"group name >  |{el.text}|")  # debug
```

Comment:
I've noticed that the similar line marked as `debug` is commented [here]( https://github.com/splunk/splunk-connect-for-snmp/pull/914/files#diff-2b38a6d07be742a0e7a28656e97622a3f8354667e8645243c08afe21245b76c3R123) . Shouldn't it also be commented here?

Reply:
commented